tests/integration/test_vulnerability_detector/test_providers/test_providers_no_os.py

```diff
@@ -57,115 +57,154 @@
     - providers
 '''
 import os
-
 import pytest
+
+from wazuh_testing.tools import configuration
+from wazuh_testing.modules.vulnerability_detector import event_monitor as evm
```

Comment:
Move this import to the end, try to group all those using alias `as`.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2547/commits/90bed077f7ebf72dd21e5d15b1af1bc674ba3704